code/modules/mob/living/simple_animal/bot/honkbot.dm

```diff
@@ -180,34 +203,57 @@
 		sensor_blink()
 		addtimer(VARSET_CALLBACK(src, spam_flag, FALSE), cooldowntimehorn)
 
+/mob/living/simple_animal/bot/honkbot/proc/cuff_callback(mob/living/carbon/C)
+	mode = BOT_ARREST
+	sleep(1 SECONDS)
```

Comment:
You want a certain delay between the "has honked you" and "starts cuffing you" message?

Reply:
Yeah, it looks better that way. Gives a moment for them to fall over, rather than starting cuffing at the same time as the big honk.